documentation/slate/source/includes/welcome.md

```diff
@@ -24,16 +24,15 @@ A fake token is used in examples below: be aware that this token is really limit
 
 First step
 ---------------
-``` shell
-#your token is in your confirmation mail. It sounds like "3b036afe-0110-4202-b9ed-99718476c2e0"
-```
+> Your token is in your confirmation mail.
```

Comment:
This is not the case anymore : you have to connect to navitia.io portal to retrieve your token, if I'm not mistaken.
@alcalyn may confirm that